src/main/java/guessingnumber/Application.java

```diff
@@ -1,7 +1,47 @@
 package guessingnumber;
 
+import java.util.Scanner;
+
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+        guessNumber();
+    }
+
+    public static void guessNumber() {
+        int answer = (int) (Math.random() * 100 + 1);
+
+        int count = 0;
+
+        boolean continueCheck = true;
+
+        Scanner input = new Scanner(System.in);
```

Comment:
알고리즘에는 input 많이 쓰실 것 같아요. 다만 도메인 흐름을 이해하기 어려우니 변수명을 구체하해보는 건 어떨까요?
```suggestion
        Scanner userInput = new Scanner(System.in);
```